driver/kubedriver.go

```diff
@@ -300,14 +271,33 @@ func writeManifest(rel *release.Release) error {
 // Credit: the logic for this function is sourced from Helm
 // https://github.com/helm/helm/blob/8ab18f7567cedffdfa5ba4d7f6abfb58efc313f8/cmd/helm/upgrade.go#L69
 // Upgrade a Kubernetes experiment to the next release
-func (kd *KubeDriver) upgrade(chartDir string, valueOpts values.Options, group string, dry bool) error {
+func (kd *KubeDriver) upgrade(chartPathOptions action.ChartPathOptions, chartName string, valueOpts values.Options, group string, dry bool) error {
 	client := action.NewUpgrade(kd.Configuration)
 	client.Namespace = kd.Namespace()
 	client.DryRun = dry
 
-	ch, vals, err := kd.getChartAndVals(chartDir, valueOpts)
+	// copy chartpathoptions
+	client.ChartPathOptions.CaFile = chartPathOptions.CaFile
```

Comment:
Could we please remove all of this? We don't even plan to document how the user is intended to use any of these options ... for now, we are better of without this.

We do need to discuss binary/image/chart signing and verification. We will revisit all of these options at that time.

Reply:
same response